src/styles/stars.scss

```diff
@@ -0,0 +1,31 @@
+.stars {
+  display: flex;
+  gap: 4px;
+  margin-bottom: 24px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__star:first-child {
+    margin-left: 0;
+  }
+}
+
+.card__stars {
```

Comment:
The `.card__stars` class should ideally be part of the `card` block styles, not the `stars` block. Consider moving this style to the appropriate SCSS file for the `card` block to maintain separation of concerns.